cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -572,6 +574,9 @@ func kubeClientGet(uc *UpstreamController, namespace string, name string, queryT
 		klog.Error(err)
 		return nil, err
 	}
+	if err == nil && obj != nil {
+		util.SetMetaType(obj.(runtime.Object))
```

Comment:
Where will use the Meta(apiversion and kind)?

Reply:
cloudhub send msg to edge successful and then create objectsync, which will use msg resource meta
https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/cloudhub/handler/messagehandler.go#L560

Reply:
And can we get the obj from lister in L563 565 567?